frameworks/Scala/finch/src/main/scala/WebServer.scala

```diff
@@ -14,25 +11,25 @@ import io.circe._
 import io.circe.generic.auto._
 import io.circe.jawn._
 import io.circe.syntax._
-import io.finch.response.EncodeResponse
 import io.circe.{Decoder, Encoder, Json}
 
 object WebServer extends App {
 
+  import io.finch.circe._
   val json = get("json") {
```

Comment:
Can we put type here `Endpoint[io.circe.Json]` and return `Json.obj("message" -> Json.string("Hello, World!"))` directly?